testdata/source/schwab_csv/test_basic/accounts.txt

```diff
@@ -5,3 +5,6 @@ Assets:Schwab:Intelligent-4321
 Assets:Schwab:Intelligent-4321:Cash
 Assets:Schwab:Intelligent-4321:FNDA
 Assets:Schwab:Intelligent-4321:SCHA
+Assets:Schwab:Intelligent-4321:SMAL040120203200P
```

Comment:
I would see some value in converting option symbol names in the same way that the Schwab OFX export used to do. Do these match previous transactions in your journal that were imported via Schwab OFX?

Reply:
I think the OFX importer does no conversion. I always had to go edit the symbol names by hand.
Perhaps a small improvement over my solution is to replace spaces with dashes, but drop the slashes.

Reply:
Oh in that case I think whatever you want to do is fine. Dashes might help readability a bit if they are allowed in beancount.

Reply:
Turns out no dashes in Beancount commodities, so simplest to strip everything.